app/services/the_tvdb_service.rb

```diff
@@ -2,6 +2,37 @@ class TheTvdbService
   BASE_URL = 'https://api.thetvdb.com'.freeze
   API_KEY = ENV['THE_TVDB_API_KEY'].freeze
 
+  # daily or weekly
+  def import!(worker_type)
```

Comment:
Why not pass `worker_type` to `def initialize()`?  Also you've leaked the concept of being part of a "worker" into this service, which isn't good; what happens if we change the intervals?  Instead, maybe try taking a reference to a set, like instead of `daily`, do `currently_airing` (you could name the variable `set_name` or something idfk)

If you do that, then you could have this for `query_data`:

```ruby
def query_data
  return unless %w[currently_airing missing_thumbnails].include? @set_name
  send(@set_name)
end
```